APIs/SentinelHub/Byoc/Examples.qmd

```diff
@@ -1,622 +1,182 @@
 ---
-title: Bring Your Own COG API
+title: BYOC API examples
 ---
 
-## Overview
-
-Bring Your Own COG API (or shortly \"BYOC\") enables you to import your
-own data in Sentinel Hub and access it just like any other data you are
-used to. To be able to do so, the following conditions should be met:
-
--   Store your raster data in the cloud optimized geotiff (COG)
-    [format](https://www.cogeo.org/){target="_blank"} on your own S3
-    bucket in the supported region.
--   Configure the bucket\'s permissions so that Sentinel Hub can read
-    them.
--   Import tiles using the Dashboard or API.
-
-Your data needs to be organized into collections of tiles. Each tile
-needs to contain a set of bands and (optionally) an acquisition date and
-time. Tiles with the same bands can be grouped into collections. Think
-of the Sentinel-2 data as a collection of Sentinel-2 tiles.
-
-#### A note about COG overviews used for processing
```

Comment:
Since it is a note , what do you think about representing it as callout using:

::: {.callout-note}
When processing data, we select the nearest overview level which has higher resolution than your request, or the full resolution image.
:::